README.md

```diff
@@ -28,7 +28,7 @@ bash build-mini.sh
 
 ## 4ku-mini Size
 ```
-3,901 bytes
+3,986 bytes
```

Comment:
There should be a typo here, 3901-5 is 3896 :)

Reply:
Thanks, fixed :)